README.md

```diff
@@ -45,6 +47,34 @@ PubsubClient.publish(message, 'some-topic') do |result|
 end
 ```
 
+### Subscribing
+
+To subscribe to a topic, a client must first get a handle to the subscriber object. After doing so, a call to `subscriber.listener` will yield two arguments: the data (most clients will only need this) and the full Pub/Sub message (for anything more robust). Optionally, a client can choose to handle exceptions raised by the subscriber.
+
+#### Example
+```ruby
+subscriber = PubsubClient.subscriber('some-topic')
+
+# Optional
+subscriber.on_error do |ex|
```

Comment:
I may be wrong about this, but it appears calling `#on_error` _before_ `#listener` will raise a `ConfigurationError`. If that is true, you should update this code and add a note about the importance of ordering these two method calls.